layouts/partials/default.html

```diff
@@ -22,6 +22,16 @@
           {{ partial "breadcrumb.html" . }}
         </div>
         <!-- end breadcrumb feature -->
+        <!-- Edit this page btn -->
+        <div class="p-lg-5 p-4 bg-white position-relative" id="content">
+          <div class="d-flex flex-row-reverse">
+            <a href="{{ .Site.Params.editPrefix}}/{{ .File.Path }}">
```

Comment:
This line causes a warning in the Hugo server:

`WARN 2022/04/05 21:58:03 .File.Path on zero object. Wrap it in if or with: {{ with .File }}{{ .Path }}{{ end }}`

Could you refactor this to resolve the new warning?

Reply:
It's just a little change i will fix it!